src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/SblInstanceEvent.cs

```diff
@@ -0,0 +1,82 @@
+using Newtonsoft.Json;
+using Storage.Interface.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Altinn.Platform.Storage.Models
+{
+    /// <summary>
+    /// Model for the instance event for interaction with the SBL solution
+    /// </summary>
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class SblInstanceEvent
```

Comment:
Seeing as this will only be used for the messagebox integration we could probably simplify this a lot to limit the amount of data being transmitted between the two platforms. Let's discuss which fields could be removed.

![image](https://user-images.githubusercontent.com/47737608/65852524-a9626000-e356-11e9-9561-0f80de9095f8.png)